admin-dev/themes/new-theme/js/pages/product/combination/combination-modal/CombinationModal.vue

```diff
@@ -281,17 +281,22 @@
         );
 
         iframeInputs.forEach((input: Element) => {
+          // @ts-ignore
```

Comment:
Why do you need a ts-ignore ?

Reply:
I think it's because there is already too wide type here:  https://github.com/tleon/PrestaShop/blob/Issue-33965-fix-next-previous-combinations/admin-dev/themes/new-theme/js/pages/product/combination/combination-modal/CombinationModal.vue#L279

This should be changed from 
```
        const iframeInputs = <NodeListOf<Element>>iframeBody.querySelectorAll(
```
to
```
        const iframeInputs = <NodeListOf<HTMLInputElement>>iframeBody.querySelectorAll(
```

Then `// @ts-ignore` and `Element` in line 283 can be removed.
![Zrzut ekranu 2023-10-22 o 22 29 09](https://github.com/PrestaShop/PrestaShop/assets/13930984/c774483e-4fe3-4614-8f9b-1a382e5e5e2c)

Then types will not lie and everything should be working without an error.
![Zrzut ekranu 2023-10-22 o 22 29 22](https://github.com/PrestaShop/PrestaShop/assets/13930984/bcbac04c-79ed-45c6-b566-90def2bf81ea)

Hope it makes sense.

Reply:
We can't force type with HTMLInputElement because the selectors doesn't only catch inputs, and for some reason checking `instanceof HTMLInputElement` was accepted by TypeScript but it didn't work when the code was executed

Reply:
Yes, you're right. I've checked the selector(`'form[name="combination_form"] input, form[name="combination_form"] textarea, form[name="combination_form"] select'` so HTML input or HTML select) and it could be like this:

![Zrzut ekranu 2023-10-24 o 13 53 45](https://github.com/PrestaShop/PrestaShop/assets/13930984/bbe4f7b5-abd2-492f-acb6-5cbf6c220ade)